<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="audioDeviceTab">
      <attribute name="title">
       <string>Audio device</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Please select an audio device:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QGroupBox" name="deviceInfoBox">
         <property name="title">
          <string>AudioDevice</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="6" column="0">
           <widget class="QCheckBox" name="depth32Chk">
            <property name="text">
             <string>32 bit (float)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QListView" name="sampleRatesList">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Channels:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Sample rates:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="depth16Chk">
            <property name="text">
             <string>16 bit</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Sample sizes:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="depth8Chk">
            <property name="text">
             <string>8 bit</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QListView" name="channelsList">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="AudioDevicesListView" name="deviceList">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="conversionTab">
      <attribute name="title">
       <string>Conversion</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Sample rate conversion</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="5" column="0">
           <widget class="QRadioButton" name="sinc_bestRadio">
            <property name="text">
             <string>Sinc best qualit&amp;y (slower, better quality)</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">sampleRateGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="sinc_fastestRadio">
            <property name="text">
             <string>Sinc fas&amp;test (default, fast, good quality)</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">sampleRateGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="zero_order_holdRadio">
            <property name="text">
             <string>&amp;Zero order hold (very fast, poor quality)</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">sampleRateGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="linearRadio">
            <property name="text">
             <string>&amp;Linear (fastest, very poor quality)</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">sampleRateGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QRadioButton" name="sinc_mediumRadio">
            <property name="text">
             <string>Sinc medium &amp;quality (slow, very good quality)</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">sampleRateGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Select the conversion method that will be used when a file has a different sample rate than the one provided by the audio device (eg. file is 44.1kHz and the audio device uses 48kHz).</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
      <zorder>groupBox</zorder>
      <zorder>verticalSpacer</zorder>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AudioDevicesListView</class>
   <extends>QListView</extends>
   <header>bigglesworth.wavetables.waveplay</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="sampleRateGroup"/>
 </buttongroups>
</ui>
